src/atoms/Flex/__tests__/Flex.web.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import View from '../../View';
+import Flex from '../index';
+
+describe('Renders <Flex /> correctly', () => {
```

Comment:
please write tests based on the convention it was discussed the other day

Reply:
added